Update profile in junqiu.md (#2990)
opensearch-project/project-website: _community_members/junqiu.md

redirect_from: '/authors/junqiu/'
---

> **Junqiu Lei** is a software development engineer at Amazon Web Services working on vector search(k-NN) and maps visualization in OpenSearch project.

Reviewer: ```suggestion
**Junqiu Lei** is a software development engineer at Amazon Web Services working on vector search (k-NN) and map visualizations for the OpenSearch Project.
```
